Tests/UnitTests/Core/Propagator/AtlasStepperTests.cpp

```diff
@@ -551,12 +551,12 @@ BOOST_AUTO_TEST_CASE(StepSize) {
   // TODO figure out why this fails and what it should be
   // BOOST_CHECK_EQUAL(stepper.overstepLimit(state), tolerance);
 
-  stepper.setStepSize(state, 5_cm);
-  BOOST_CHECK_EQUAL(state.previousStepSize, navDir * stepSize);
-  BOOST_CHECK_EQUAL(state.stepSize.value(), 5_cm);
+  stepper.setStepSize(state, 5_mm);
```

Comment:
Is the unit switch just cleanup?

Reply:
don't remember but I guess this is arbitrary. will try to revert